### DISCUSSION TITLE
Would it be possible to build an ERCF with 24 colours ?

### DISCUSSION THREAD

**Discussion by Yveske1966** [Enraged-Rabbit-Community/ERCF_v2#137] [Q&A] [answered]
Wanting to take up HueForge printing but without continuously swapping colours

**Comment by norris108** [comment, marked as answer]
It's _possible_? Yes, but is it _recommended_?

The recommendation is 8 or 12 gates, and that's what most, if not all, kits provide

Anything over 12 and you'll be pretty much on your own to self source

You'll need good quality components, such as the 8mm linear rods, d-shaft, etc., and at the length a 24 gate ERCF will be, those components would need to be 100% perfectly straight and very strong to prevent them from bending/bowing

**Reply by Yveske1966** [reply]
Thanks !!!

**Reply by old-relic** [reply]
Hi, I made a 15 gate version. It works very well. The biggest difficulty I ran into was not being able to find d-rod for the selector drive gears. I had to use drill rod (silver steel rod) and make flats for the selector gear set screws with a Dremel wheel. I think 15 gates might be the practical limit without beefing up the selector guide rods. My setup shows some flexing when the servo picks the centre gates. FWIW
![15gate ERCF](https://github.com/user-attachments/assets/faf92241-9529-47b5-ac36-6b905f73a8fb)
